packages/reports/addon/templates/components/navi-request-preview.hbs

```diff
@@ -40,7 +40,7 @@
           </th>
         {{/each}}
       </thead>
-      <tbody class="navi-request-preview__table-body">
+      <tbody class={{concat "navi-request-preview__table-body" (if this.editingColumn " navi-request-preview__table-body--editing")}}>
```

Comment:
don't have to use `concat` 
`class="navi-request-preview__table-body{{if this.editingColumn " navi-request-preview__table-body--editing"}}"`